### PR TITLE
Cleaned up strategy file.

### PR DESCRIPTION
**Pull request by mukundv-chrome**
Got rid of redundant list in file and renamed lists since this file has strategies for multiple engines.

### CODE REVIEW

File: src/python/fuzzing/strategy.py [hunk 1]
```diff
@@ -87,6 +65,23 @@
     CORPUS_SUBSET_STRATEGY,
 ]
 
+# Lists of prefix and boolean strategies maintained for libFuzzer stats.
+libfuzzer_strategies_with_prefix_value = [
```
Comment (Dor1s) [inline src/python/fuzzing/strategy.py]: can you capitalize these two as well? this way all the names will be consistent inside this file

Comment (inferno-chromium) [reply src/python/fuzzing/strategy.py]: +1

**Pull request merged**